companies/lewde.json

```diff
@@ -0,0 +1,30 @@
+{
+    "slug": "lewde",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "utility"
+    ],
+    "name": "Lechwerke AG",
+    "address": "Schaezlerstraße 3\n86150 Augsburg\nDeutschland",
+    "phone": "08213280",
```

Comment:
The phone and fax numbers need to be in the international format and without any characters other than numbers, `+` and space (so `+49 821 328 1170` for the fax for example).

If you have used the formatter in the JSON generator, there is unfortunately a bug in the library we use which causes the unformatted number to be exported even though the correct one is shown in the generator. Sorry about that.